### ISSUE TITLE
Accessibility Feedback - Usability Testing Prep: GI Bill Comparison Tool - Ratings

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#15239]
**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Should
* At least 1 tester should be using a phone if possible. The research might reveal some things that desktop might not.

## Consider
* The mini-accordions don't have a hover state. It will be good to note if users understand these can be expanded by clicking or tapping.

## Definition of Done
- [x] Team comments on ticket that they've received feedback and VSP has answered any questions
- [ ] Team can close the ticket after commenting

**Comment by amyknox** [comment]
Thanks for the feedback @1Copenut .  We've updated our testing plan to recruit people to test on phone.  We'll work on a hover state for the ratings - Allison suggested that modification, too, so extra incentive to address it.

**Comment by amyknox** [comment]
@JenLJ - can we add "how do users understand these can be expanded by clicking or tapping."  to the "Do users understand the UI & supporting content?" research question?

**Comment by 1Copenut** [comment]
Thank you @amyknox . Glad the feedback was useful! Feel free to close this issue as you're ready.

**Comment by JenLJ** [comment]
@amyknox - I added "Do users understand accordions can be expanded by clicking or tapping?" to the "Do users understand the UI and supporting content?" section of the research plan
